.papr.yml

```diff
@@ -15,6 +15,7 @@ timeout: 45m
 tests:
   -  sh .papr_prepare.sh
 
+context: "Fedora fedora/27/atomic"
```

Comment:
Maybe just `FAH27`?  That's how I tend to spell it.

Reply:
OK